data/yaml/venues/caldpseudo.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: caldpseudo
```

Comment:
The acronym field should have the workshop's preferred formatting (`CALD-pseudo`)